graphql-schema/schema.graphql

```diff
@@ -214,6 +215,14 @@ type Group {
     users: [User]
 }
 
+type PastGroups {
```

Comment:
I am thinking a better way of doing this is just adding a `year` field to the `Group` type. Adding a new type might not be required.